docs/how-to/create_plugin.rst

```diff
@@ -0,0 +1,100 @@
+.. _how_to_create_plugin:
+
+How to create a plugin
+======================
+
+A new tool or programming language may be easier to use craft-parts with the help of a new plugin. This document contains instructions on how to recreate the simple :py:class:`~craft-parts.plugins.dump_plugin.DumpPlugin` plugin.
```

Comment:
this first sentence reads awkwardly to me

Reply:
Jeez, not sure how I missed this in my first proofread. Thanks!